lib/backend/k8s/k8s_test.go

```diff
@@ -1967,6 +1987,154 @@ var _ = testutils.E2eDatastoreDescribe("Test Watch support", testutils.Datastore
 		ctx = context.Background()
 	})
 
+	It("Should support watching Profiles", func() {
+		By("Watching a single Profile", func() {
+			name := "default"
+			watch, err := c.Watch(ctx, model.ResourceListOptions{Name: name, Kind: apiv3.KindProfile}, "")
+			Expect(err).NotTo(HaveOccurred())
+			event := ExpectAddedEvent(watch.ResultChan())
+			Expect(event.New.Key.String()).To(Equal("Profile(kns.default)"))
+		})
+		By("Watching all Profiles", func() {
+			watch, err := c.Watch(ctx, model.ResourceListOptions{Kind: apiv3.KindProfile}, "")
+			Expect(err).NotTo(HaveOccurred())
+			ExpectAddedEvent(watch.ResultChan())
```

Comment:
We generate one of these per namespace, so might want to verify we get a second one here too.